lib/embeds/instagram.js

```diff
@@ -18,7 +18,16 @@ export default {
       : '';
 
     const script = `<script src="${protocol}//platform.instagram.com/en_US/embeds.js"></script>
-      <script>resize()</script>`;
+      <script>
+        const interval = setInterval(function () {
+          const iframe = document.querySelector('iframe.instagram-media-rendered');
```

Comment:
i would add a unique id to the classname and put it in this script so that we're sure it's selecting the right iframe